FunColor.cs

```diff
@@ -8,4 +8,6 @@ public static class FunColor
     public static Color DotNetPurple2024 => Color.FromUint(0xFF512BD4);
 
     public static Color XamarinBlue2011 => Color.FromUint(0xFF5596D8);
+
+    public static Color BratSummer => Color.FromUint(0xFF8ACE00);

```

Comment:
```suggestion
    public static Color BratAutumn => Color.FromUint(0xFFFF8C00);

```

https://x.com/charli_xcx/status/1830695037925237025

Reply:
RIP. in peace brat summer. thank you for this beautiful work

Reply:
I'm keeping both though